src/scripts/main.js

```diff
@@ -1 +1,83 @@
 'use strict';
+
+// #region  Promise 1
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)